Update sb_replace_files methods

lindsayplatt opened the pull request:
Since this repo has been created, we have updated the fxn `sb_replace_files` (see https://github.com/USGS-R/slim-data-release-template/pull/3). This new pattern is being used in https://github.com/USGS-R/mntoha-data-release and so I tried to follow that repo as an example to update this one.

lindsayplatt committed:
fix incomplete final line warning

lindsayplatt committed:
pull jread's updates since I forked the repo

Merge branch 'release' of github.com:jread-usgs/hyperscales-data-release into release

# Conflicts:
#	4_inputs.yml

lindsayplatt committed:
use update sb_replace_files fxn from slim-data-release-template & mntoha-data-release

lindsayplatt committed:
this was accidentally committed (needed in a different branch)

lindsayplatt committed:
I don't want there to be any changes to this file in the current PR

Commit:
updates with earlier modeling work

jordansread commented on remake.yml:
```diff
@@ -32,90 +32,105 @@ sources:
 targets:
```
I think you need to to 
```yaml
packages:
  - scipiper
```

because of the use of the `create_task_plan()` and other scipiper functions

lindsayplatt replied:
I read back over to see if there were other packages I missed. `remake` is used in `verify_uploads`, so I added that as well.

jordansread reviewed (commented):
I am running all of this to see if there are any snags (found a small one), but that will take a while. So will come back and comment after the rebuilds are complete.

lindsayplatt committed:
scipiper & remake pkgs are needed

jread-usgs commented:
Did some other things to deal with this build, as there were some simple and small bugs I needed to get around (e.g., changes to `lubridate` that caused an error). I also had moved a ton of these large files off of my laptop onto an external and had to wrestle with trying to free up enough space to bring them back to make this pipeline work. Definitely looking forward to a Yeti-based way to do that in the future!

Anyhow, it is building/uploading now. Still WIP

Commit:
log files from build

Merge branch 'update_sbreplacefiles_methods' of https://github.com/lindsayplatt/hyperscales-data-release into lplatt

# Conflicts:
#	remake.yml

Commit:
merge from Lindsay

Commit:
need this update

jread-usgs commented:
This is good to go but see my PR on your branch

lindsayplatt committed:
Merge pull request #1 from jread-usgs/lplatt

My minor changes to your PR to get the build right

lindsayplatt commented:
Merged that in! I cannot merge because it is your repo :)
